src/sentry/search/snuba/backend.py

```diff
@@ -353,8 +362,12 @@ def do_search(project_id, environment_id, tags, start, end,
                 group_data[group_id] = defaultdict(list)
 
             dest = group_data[group_id]
-            for k, v in obj.items():
-                dest[k].append(v)
+            if len(required_aggregations) == 1:
+                alias = list(required_aggregations)[0]
+                dest[alias].append(obj)
+            else:
+                for k, v in obj.items():
+                    dest[k].append(v)
```

Comment:
I'm tempted to change how Snuba works here because I think dealing with this on the response side kinda sucks.